src/background.ts

```diff
@@ -1433,10 +1505,62 @@ app.on("ready", async () => {
     }
   }
 
+  // runEngineAllの前にVVPPを読み込む
+  let filePath: string | undefined;
+  if (process.platform === "darwin") {
+    filePath = filePathOnMac;
+  } else {
+    if (process.argv.length > 1) {
+      filePath = process.argv[1];
+    }
+  }
+
+  // 多重起動防止
+  if (
+    // !isDevelopment &&
+    !app.requestSingleInstanceLock({
+      filePath,
+    } as SingleInstanceLockData)
+  ) {
+    log.info("VOICEVOX already running. Cancelling launch.");
+    log.info(`File path sent: ${filePath}`);
+    app.quit();
+    return;
+  }
+
   createWindow().then(() => runEngineAll());
 });
 
-app.on("second-instance", () => {
+app.on("second-instance", async (event, argv, workDir, rawData) => {
```

Comment:
`second-instance`が何なのか、たぶん僕と名無し。さん以外だとあまり直感的じゃない雰囲気を感じました。
ちょっと説明書くのどうでしょう。こんな程度で。

```suggestion
// 他のプロセスが起動したとき、`requestSingleInstanceLock`経由で`rawData`が送信される。
app.on("second-instance", async (event, argv, workDir, rawData) => {
```